Clicker/SectionControllers/EmptyStateSectionController.swift

```diff
@@ -23,6 +28,10 @@ class EmptyStateSectionController: ListSectionController {
     
     override func cellForItem(at index: Int) -> UICollectionViewCell {
         let cell = collectionContext?.dequeueReusableCell(of: EmptyStateCell.self, for: self, at: index) as! EmptyStateCell
+        cell.session = session
+        cell.userRole = userRole
+        cell.nameViewDelegate = nameViewDelegate
```

Comment:
instead of assigning them directly here, we should either
1) make an `EmptyStateCellDelegate` protocol which has `session`, `userRole`, and `nameViewDelegate`
2) or have a `configure(with: session: Session, userRole: UserRole, nameViewDelegate: NameViewDelegate)` function in EmptyStateCell

Reply:
I will clean up the places where I did this.